CMakeLists.txt

```diff
@@ -20,49 +20,38 @@ else()
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic")
 endif()
 
-# Define macro for setting cxx11 as standard for building a target.
-macro(use_cxx11 target)
-  if ((CMAKE_VERSION VERSION_LESS "3.2") AND MACOSX)
-    # Because CXX_STANDARD is not supported for OSX in Cmake 3.1.
-    target_compile_options(${target} PRIVATE "-std=c++11")
-  else()
-    set_target_properties(${target} PROPERTIES
-      CXX_STANDARD 11
-      CXX_STANDARD_REQUIRED ON
-      CXX_EXENSIONS NO)
-  endif()
-endmacro(use_cxx11)
-
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
What do you say we rather go with
```
set(CMAKE_CXX_STANDARD 11)
set(CMAKE_CXX_STANDARD_REQUIRED ON)
set(CMAKE_CXX_EXENSIONS NO)
```

Reply:
Btw, I just realized I had a type in my old code! I had `EXENSIONS` instead of `EXTENSIONS`!